scripts/gha/test_lab.py

```diff
@@ -38,11 +38,7 @@
 
 """
 
-import datetime
```

Comment:
This file needs a copyright & license header as well.

Reply:
Whoopsie